src/main/java/nl/novi/techiteasy1121/services/TelevisionService.java

```diff
@@ -1,72 +1,88 @@
 package nl.novi.techiteasy1121.services;
```

Comment:
De methoden in je services zijn goed gestructureerd en duidelijk. Het gebruik van transfer methodes is een efficiënte manier om DTO's en modellen om te zetten. Blijf deze aanpak volgen.